runners/samza/src/main/java/org/apache/beam/runners/samza/runtime/KeyedTimerData.java

```diff
@@ -170,13 +170,13 @@ public void encode(KeyedTimerData<K> value, OutputStream outStream)
       }
 
       STRING_CODER.encode(timer.getTimerFamilyId(), outStream);
+      INSTANT_CODER.encode(timer.getOutputTimestamp(), outStream);
```

Comment:
Ideally, this field should be put immediately after "timestamp" field so that when perform a range query, the earliest entries will be processed first.  The assumption for this fix is that the outputTimestamp will not contribute to the order of range query.